backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/textfield/BpkTextField.kt

```diff
@@ -45,6 +50,7 @@ fun BpkTextField(
     keyboardActions: KeyboardActions = KeyboardActions(),
     maxLines: Int = 1,
     interactionSource: MutableInteractionSource = remember { MutableInteractionSource() },
+    clearAction: BpkClearAction? = null,
```

Comment:
Do we have this parameter in Figma? Does it uses the same name and type parameter?